tests/datalab/datalab/test_datalab.py

```diff
@@ -18,7 +18,6 @@
 import contextlib
 import io
 import os
-import pickle
 import timeit
 from pathlib import Path
 from unittest.mock import MagicMock, Mock, patch
```

Comment:
CI is complaining about imports that are no longer used:  https://github.com/cleanlab/cleanlab/actions/runs/12554401936/job/35137671038?pr=1222

<img width="742" alt="Screenshot 2025-01-03 at 7 07 01 PM" src="https://github.com/user-attachments/assets/5c27ab84-cec9-4fa5-bc2b-d41fbbd20f16" />